xarray/backends/common.py

```diff
@@ -374,6 +374,20 @@ class BackendEntrypoint:
     open_dataset_parameters: tuple | None = None
     """list of ``open_dataset`` method parameters"""
 
+    @property
+    def backend_description(description: str | None):
```

Comment:
I feel like the word "backend" is redundant in these method names, calling just `BackendEntrypoint.description` seems fine to me.